backend/clubs/management/commands/once_batch_remove_tags.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from clubs.models import Club, School, StudentType, Year
+
+
+class Command(BaseCommand):
+    help = (
+        "Remove target tags from orgs which selected all target tags: "
+        "target_year (Degree Type), target_school (Target School), "
+        "student_types (Students Type). This script should only be run "
+        "once on the Hub@Penn database. This script should be deleted "
+        "after it has been run once."
+    )
+    web_execute = True
+
+    def handle(self, *args, **kwargs):
+        target_years_count = Year.objects.distinct().count()
```

Comment:
Distinctness is not actually relevant here right? Otherwise could mess with the way you filter by `target_years_count` and such.

Should we delete this script or just mention that it's shelved somewhere as a comment/wherever? Could be useful sometime in the future

LGTM otherwise!

Reply:
Ok removed the `distinct()` from the query. Thanks king!